src/server/scripts/Spells/spell_holiday.cpp

```diff
@@ -824,6 +824,60 @@ class spell_brewfest_barker_bunny : public SpellScriptLoader
         }
 };
 
+enum TorchSpells
+{
+    SPELL_TORCH_TOSSING_TRAINING                    = 45716,
+    SPELL_TORCH_TOSSING_PRACTICE                    = 46630,
+    SPELL_TORCH_TOSSING_TRAINING_SUCCESS_ALLIANCE   = 45719,
+    SPELL_TORCH_TOSSING_TRAINING_SUCCESS_HORDE      = 46651,
+    SPELL_BRAZIERS_HIT                              = 45724
+};
+
+// 45724 - Braziers Hit!
+class spell_midsummer_braziers_hit : public SpellScriptLoader
+{
+    public:
+        spell_midsummer_braziers_hit() : SpellScriptLoader("spell_midsummer_braziers_hit") { }
+
+        class spell_midsummer_braziers_hit_AuraScript : public AuraScript
+        {
+            PrepareAuraScript(spell_midsummer_braziers_hit_AuraScript);
+
+            bool Validate(SpellInfo const* /*spellInfo*/) override
+            {
+                if (!sSpellMgr->GetSpellInfo(SPELL_BRAZIERS_HIT))
```

Comment:
you have to validate spells used in the script and not the spell bound to this script itself